cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -173,7 +174,8 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
         val repository = artifactory.repository(repoName)
         try {
             repository.delete(releaseName_build)
-        } catch (_: Exception) {
+        } catch (e: Exception) {
+            logger.error("Failed to delete folder: $releaseName_build", e)
```

To fix the SonarQube violation caused by the empty catch block, I added a logging statement inside the catch block to log any exceptions that occur during the deletion of the folder. This utilizes the logger, assumed to be a properly configured logging framework, to record the failure and exception details. This change makes the block non-empty, thereby improving maintainability by not silently swallowing exceptions.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)